### PR TITLE
awscli2: 2.0.54 -> 2.0.61

### PR DESCRIPTION
**Pull request by dbirks**
###### Motivation for this change

Updating to the version released today.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/admin/awscli2/default.nix [hunk 1]
```diff
@@ -29,13 +29,13 @@ let
 in
 with py.pkgs; buildPythonApplication rec {
   pname = "awscli2";
-  version = "2.0.54"; # N.B: if you change this, change botocore to a matching version too
+  version = "2.0.61"; # N.B: if you change this, change botocore to a matching version too
```
Comment (SuperSandro2000) [inline pkgs/tools/admin/awscli2/default.nix:32]: Shouldn't botcore also be updated?

Comment (dbirks) [reply pkgs/tools/admin/awscli2/default.nix:32]: I think botocore is still on a 1.x.x release ([link here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/botocore/default.nix#L15)) to match the current awscli v1. 

> N.B: if you change this, change botocore to a matching version too

I believe this is a note to change the version in the override at the top, to match what v2 of the aws-cli currently requires ([link](https://github.com/aws/aws-cli/blob/v2/setup.cfg#L7)) :v:

Comment (SuperSandro2000) [reply pkgs/tools/admin/awscli2/default.nix:32]: Can we maybe clarify this that the overwrite in the top is meant?

**Comment by arcnmx** [comment]
Looks like colorama needs overriding to <0.4.4, also it looks like there's a 2.1 version now

**Comment by SuperSandro2000** [comment]
Superseded by #104453. Closing.

**Pull request closed without merging**